pkgs/by-name/ed/edopro/package.nix

```diff
@@ -0,0 +1,262 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, premake5
+, writeShellScriptBin
+, runCommandLocal
+, symlinkJoin
+, imagemagick
+, bzip2
+, curl
+, flac
+, fmt_9
+, freetype
+, libevent
+, libgit2
+, libGL
+, libGLU
+, libjpeg
+, libpng
+, libvorbis
+, libX11
+, libxkbcommon
+, libXxf86vm
+, lua5_3
+, nlohmann_json
+, noto-fonts-cjk-sans
+, openal
+, SDL2
+, sqlite
+, wayland
+, egl-wayland
+,
+}:
+let
+  edopro-version = "40.1.4";
+
+  # the assets repo has a font instead
+  font = "${noto-fonts-cjk-sans}/share/fonts/opentype/noto-cjk/NotoSansCJK-VF.otf.ttc";
+
+  covers_url = "";
+  fields_url = "";
+  pics_url = "http://images.ygoprodeck.com/images/cards/{}.jpg";
```

Comment:
I would assume most users want higher quality card art, so I did set that. But if you do get banned you can apply the an override now.

Reply:
Let's add something about the URL format and the fact this can also be configured in `system.conf` so you don't need to rebuild the package just to change rhe download url?